velox/functions/prestosql/window/tests/WindowFunctionRegTest.cpp

```diff
@@ -16,12 +16,24 @@
 
```

Comment:
You could move this file into velox/exec/tests as well.

Reply:
Thanks for the suggestion, moved to the tests in this file to WindowFunctionRegistryTest.cpp in velox/exec/tests. Please let me know if this is fine or whether these tests should be kept separate.